config/routes.rb

```diff
@@ -3,8 +3,8 @@
   devise_for :users, controllers: {registrations: 'users/registrations', sessions: 'users/sessions'}
   get 'welcome/index'
   get '/targets/load_create_target' => 'targets#load_create_target', as: :load_create_target
+  get '/targets/load_delete_target' => 'targets#load_delete_target', as: :load_delete_target
```

Comment:
I would add these routes as collections inside target resource. But, looking at what `load_delete_target` does, I think it would be better to make this route a member one, so the final route is `/targets/deleted/:id`, this route is more restful.